### ISSUE TITLE
[Accessibility- Form C11.1] The filter field doesn't have a label

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3827]
### Problem Statement

Environment: Tribe

#### Steps to Reproduce
Access to app center in extended mode

#### Current behavior:

The filter field doesn't have a label

#### Expected behavior:

Add a title attribute to the filter field as the placeholder

- title=" Filter by title or description"

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
